Engine Hacks/SkillSystem/Internals/CalcLoops/WeaponUsabilityLoop/WeaponLockCalcLoop.event

```diff
@@ -0,0 +1,42 @@
+PUSH
+ORG 0x16574
+#incbin "CanUnitWieldWeapon.dmp"
+POIN ItemTable
+POIN WLockLookupList
+POIN WeaponLockLoop
+POP
+//vanilla function size: 0x1DA
+//new function size: 0x100
+//saved: 218 bytes
+
+ALIGN 4
+WLockLookupList:
+//WORD 0x003C1C00 0xF0070000
+WORD 0x00000400 0x00040000 	//Monster Lock
+WORD 0x00000800 0x00010000	//Roy Lock
+WORD 0x00001000 0x00020000	//Myrm Lock
+WORD 0x00010000 0xFEFEFEFE  //Unusable
```

Comment:
0xFEFEFEFE is a bitfield state that's realistically never going to be true (all char/class abilities sans Mounted Aid/Promoted/Roy Lock/Don't Give EXP), and since it's the Unusable bit it if it is set on a weapon it should compare against something that reliably always returns false; I would've used 0xFFFFFFFF but it's the list terminator and I wouldn't want to break things accidentally with it

Reply:
this actually could probably just be something like all 3 aid icon display bits but better safe than sorry

Reply:
ok but
you are using a `tst`, which is bitwise and, and checking if the result is (non-)zero (using `beq`). If you're using `0xFEFEFEFE` as value, then the test will succeed if *any* of the corresponding bits of the cattr is set (and since you have all but 4 bits set that's most bits).

Reply:
OH